src/components/TokenSelection/sort.ts

```diff
@@ -25,3 +25,27 @@ export function sortTokensBySymbol(tokens: TokenInfo[]) {
     a.symbol.toLocaleLowerCase() < b.symbol.toLocaleLowerCase() ? -1 : 1
   );
 }
+
+function getTokenBalance(token: TokenInfo, balances: BalancesState): number {
+  return parseFloat(
+    formatUnits(balances.values[token.address]!, token.decimals)
+  );
+}
+
```

Comment:
This is where the undefined BigNumber error gets triggered, causing an app crash

Reply:
Fixed, nice find.